inst/doc/geoknife.R

```diff
@@ -4,6 +4,9 @@ options(continue=" ")
 options(width=60)
 library(knitr)
 library(geoknife)
+query <- geoknife::query
+`values<-` <- geoknife::`values<-`
+id <- geoknife::id
```

Comment:
are you defining these because otherwise they get masked by same-named functions from other packages? what happens if you leave out these lines?